install/gce-installation-guide.md

```diff
@@ -0,0 +1,119 @@
+
+### Creating the Image
+
+The following (substituing your preferred disribution project and image family) will produce an image with nested virtualization enabled in your currently active GCE project:
+
+```bash
+SOURCE_IMAGE_PROJECT=ubuntu-os-cloud
+SOURCE_IMAGE_FAMILY=ubuntu-1804-lts
+IMAGE_NAME=${SOURCE_IMAGE_FAMILY}-nested
+
+gcloud compute images create \
+    --source-image-project $SOURCE_IMAGE_PROJECT \
+    --source-image-family $SOURCE_IMAGE_FAMILY \
+    --licenses=https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx \
+    $IMAGE_NAME
+```
+
+`gcloud` should report that the image was created. You can verify that the image has the nested virtualization license with `gcloud compute images show $IMAGE_NAME`. This should produce output like the following (some fields have been removed for clarity and to redact personal info):
+
+    diskSizeGb: '10'
+    kind: compute#image
+    licenseCodes:
+    - '1002001'
+    - '5926592092274602096'
+    licenses:
+    - https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx
+    - https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/licenses/ubuntu-1804-lts
+    name: ubuntu-1804-lts-nested
+    sourceImage: https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-1804-bionic-v20180522
+    sourceImageId: '3280575157699667619'
+    sourceType: RAW
+    status: READY
+
+The primary criterion of interest here is the presence of the "enable-vmx" license. Without that licence Kata will not work. The presence of that license instructions the Google Compute Engine hypervisor to enable Intel's VT-x instructions in virtual machines created from the image. Note that nested virtualizaiton is only available in VMs running on Intel Haswell or later CPU microarchitectures.
+
+### Verifying VMX is Enabled
+
+Assuming you created a nested-enabled image using the instructions above you can verify that VMs created from this image are VMX-enabled with the following:
+
+1. Create a VM from the image created above
+
+    ```bash
+    gcloud compute instances create \
+        --image $IMAGE_NAME \
+        --machine-type n1-standard-2 \
+        --min-cpu-platform "Intel Broadwell" \
+        kata-testing
+    ```
+
+> **NOTE**: In most zones the `--min-cpu-platform` argument can be omitted. It is only necessary in GCE Zones that include hosts based on Intel's Ivybridge platform.
+
+2. Verify that it has the VMX CPUID flag set
+
+    ```bash
+    gcloud compute ssh kata-testing
+    
+    lscpu |grep vmx >/dev/null 2>&1 && echo "VMX Enabled! :)" || echo "No VMX :("
```

Comment:
I wonder if it's worth replacing this with a call to `kata-runtime kata-check` (ideally via `sudo` so it performs all possible checks)? That would be simpler to type but wouldn't be checking for the Intel virtualisation extensions explicitly - it would be checking for the virtualisation extensions for the platform you happen to be running on :)

Unless we call `kata-check` here and have another pre-req test at the top to ensure we're on the right type of system maybe?

```
$ [ -z "$(lscpu|grep GenuineIntel)" ] && { echo "ERROR: Need an Intel CPU"; exit 1; }
```